build_tools/travis/install.sh

```diff
@@ -37,22 +37,25 @@ if [[ "$DISTRIB" == "conda" ]]; then
     export PATH=$MINICONDA_PATH/bin:$PATH
     conda update --yes conda
 
-    # Configure the conda environment and put it in the path using the
-    # provided versions
+    TO_INSTALL="python=$PYTHON_VERSION pip pytest pytest-cov \
+                numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
+                cython=$CYTHON_VERSION"
+
     if [[ "$INSTALL_MKL" == "true" ]]; then
-        conda create -n testenv --yes python=$PYTHON_VERSION pip \
-            pytest pytest-cov numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
-            mkl cython=$CYTHON_VERSION \
-            ${PANDAS_VERSION+pandas=$PANDAS_VERSION}
-            
-    else
-        conda create -n testenv --yes python=$PYTHON_VERSION pip \
-            pytest pytest-cov numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION \
-            nomkl cython=$CYTHON_VERSION \
-            ${PANDAS_VERSION+pandas=$PANDAS_VERSION}
+        TO_INSTALL="$TO_INSTALL mkl"
+    fi
```

Comment:
here you need a else clause with nomkl.

Reply:
@lesteve I do not understand why. Doesn't conda automatically install nomk version?
Do I assume correctly you mean this?
```
    if [[ "$INSTALL_MKL" == "true" ]]; then
        TO_INSTALL="$TO_INSTALL mkl"     
    else
        TO_INSTALL="$TO_INSTALL nomkl"
    fi
```

Reply:
Found out why :) Change done. Well spotted.